tests/buy_button_test.py

```diff
@@ -0,0 +1,151 @@
+import json
+import unittest
+import os
+import sys
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+from server import db
+from sqlalchemy.sql import func
+from server.utils.database_test import DatabaseTest
+from server.models import Transaction, Item, Store
+
+KEY_SUCCESS = "success"
+KEY_MESSAGE = "message"
+KEY_ERROR = "error"
+KEY_ID = "id"
+KEY_QUANTITY = "quantity"
+KEY_EXPECTED = "expected"
+KEY_PARAMS = "parameters"
+KEY_USER_ID = "user_id"
+KEY_AMOUNT = "AMOUNT"
+
+class BuyButtonTest(DatabaseTest):
+    def setUp(self):
+        super().setUp()
+        self.user1_id = 1
+        self.user2_id = 2
+        self.user3_id = 3
+        trans1 = Transaction (
+            user_id=self.user1_id,
+            ticket_amount=1500,
+            activity="Transaction Test",
```

Comment:
Not a transaction test. Maybe buy test?

Reply:
Its still a transaction so I'll keep as "Transaction Test"